packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -151,6 +151,11 @@ export class RemoteMediaGroup {
     throw new Error(`remote media object ${remoteMedia.id} not found in the group`);
   }
 
+  public setPreferLiveVideo(preferLiveVideo: boolean) {
```

Comment:
May be rename this to updatePreferLiveVideoOption

Reply:
I think I prefer existing name `setPreferLiveVideo`